docs/cli-reference/start.md

```diff
@@ -15,27 +15,39 @@ gpustack start [OPTIONS]
+| `--ray-dashboard-agent-grpc-port` value     | `40101`                                | Port for Ray dashboard agent gPRC listen. Used when Ray is enabled.                                                                   |
+| `--ray-dashboard-agent-listen-port` value   | `52365`                                | Port for Ray dashboard agent HTTP listen. Used when Ray is enabled.                                                                   |
+| `--ray-metrics-export-port` value           | `40103`                                | Port for Ray metrics export. Used when Ray is enabled.                                                                                |
+| `--bin-dir` value                           | (empty)                                | Directory to store additional binaries, e.g., versioned backend executables.                                                          |
+| `--pipx-path` value                         | (empty)                                | Path to the pipx executable, used to install versioned backends.                                                                      |
+| `--system-default-container-registry` value | `docker.io`                            | Default container registry for GPUStack to pull system images.                                                                        |
+| `--image-name-override` value               | (empty)                                | Override the default image name for the GPUStack container.                                                                           |
+| `--image-repo` value                        | `gpustack/gpustack`                    | Override the default image repository for the GPUStack container.                                                                     |
+| `--gateway-mode` value                      | `auto`                                 | Gateway mode to run in: embedded, incluster, external, disabled, or auto (default).                                                   |
+| `--gateway-kubeconfig` value                | (empty)                                | Path to the kubeconfig file for gateway. Only useful for external gateway-mode.                                                       |
+| `--gateway-gateway-concurrency` value       | `16`                                   | Number of concurrent connections for the gateway.                                                                                     |
+| `--service-discovery-name` value            | (empty)                                | The name of the service discovery service in DNS. Only useful when deployed in Kubernetes with service discovery.                     |
+| `--namespace` value                         | `higress-system`                       | Kubernetes namespace for the gateway, only used when gateway_mode is 'embedded', 'incluster' or 'external'.                           |
```

Comment:
Should it be `--gateway-namespace`

Reply:
I would like to reuse this parameter parsing to gpustack/runtime to start model instance pod.
I would update the description for this parameters and add logic to reuse this parameters for runtime